### DISCUSSION TITLE
Published: Development and Evaluation of an In Silico Dermal Absorption Model Relevant for Children

### DISCUSSION THREAD

**Discussion by abdullahhamadeh** [Open-Systems-Pharmacology/Forum#1079] [Announcement]
The dermal modeling group at the University of Waterloo are excited to announce the publication of the following paper, which models skin penetration in children using the [OSP Skin Permeation Model](https://github.com/Open-Systems-Pharmacology/Skin-permeation-model).

[**Development and Evaluation of an In Silico Dermal Absorption Model Relevant for Children**
by Yejin Esther Yun, Daniella Calderon-Nieva, Abdullah Hamadeh and Andrea N. Edginton
School of Pharmacy, University of Waterloo, Waterloo, ON N2G 1C5, Canada](https://www.mdpi.com/1999-4923/14/1/172)

**Abstract**
The higher skin surface area to body weight ratio in children and the prematurity of skin in neonates may lead to higher chemical exposure as compared to adults. The objectives of this study were: (i) to provide a comprehensive review of the age-dependent anatomical and physiological changes in pediatric skin, and (ii) to construct and evaluate an age-dependent pediatric dermal absorption model. A comprehensive review was conducted to gather data quantifying the differences in the anatomy and physiology of child and adult skin. Maturation functions were developed for model parameters that were found to be age-dependent. A pediatric dermal absorption model was constructed by updating a MoBi implementation of the Dancik et al. 2013 skin permeation model with these maturation functions. Using a workflow for adult-to-child model extrapolation, the predictive performance of the model was evaluated by comparing its predicted rates of flux of diamorphine, phenobarbital and buprenorphine against experimental observations using neonatal skin. For diamorphine and phenobarbital, the model provided reasonable predictions. The ratios of predicted:observed flux in neonates for diamorphine ranged from 0.55 to 1.40. For phenobarbital, the ratios ranged from 0.93 to 1.26. For buprenorphine, the model showed acceptable predictive performance. Overall, the physiologically based pediatric dermal absorption model demonstrated satisfactory prediction accuracy. The prediction of dermal absorption in neonates using a model-based approach will be useful for both drug development and human health risk assessment.

**Comment by StephanSchaller** [comment]
@abdullahhamadeh , great! Congratulations. Can you please also post this[ here](https://github.com/Open-Systems-Pharmacology/OSP-based-publications-and-content/issues), and if you send me a quick line, I can post it on the OSP LinkedIn Page!
